style.css

```diff
@@ -0,0 +1,153 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  scroll-behavior: smooth;
+}
+
+@media screen and (min-width: 374px) {
+  body {
+    background-image: url(background-1.svg);
+    background-position: center;
+    font-weight: 700;
+    font-size: 18px;
+    color: var(--primary);
+    animation: logo 2s ease-in-out;
+    text-decoration: none;
+  }
```

Comment:
- It will be better if you removed the `background-image: url(background-1.svg); ` from the body of your portfolio because it has caused it to appear on the entire website. It will be better if used it in your entire header section. 👍🏽 

This is what I can see 👇🏽 
| The background image in your portfolio |
| :---:   | 
|<img width="397" alt="Screenshot 2023-02-16 at 11 23 32" src="https://user-images.githubusercontent.com/99246359/219441121-102fda1c-aa25-4f0a-996d-b18aa62c344e.png">  |

Reply:
thanks for reviewing my code.